DoctorWho/DoctorWho.Db/Repositories/EpisodeCompanionRepository.cs

```diff
@@ -0,0 +1,21 @@
+﻿
+namespace DoctorWho.Db.Repositories
+{
+    public class EpisodeCompanionRepository
+    {
+        private DoctorWhoCoreDbContext _context;
+        public EpisodeCompanionRepository(DoctorWhoCoreDbContext context)
+        {
+            _context = context;
+        }
+        public void AddCompanionToEpisode(int episodeId, int companionId)
+        {
+            var episode = _context.Episodes.Find(episodeId);
+            if (episode != null)
+            {
+                episode.EpisodeCompanions.Add(new EpisodeCompanion { CompanionId = companionId, EpisodeId = episodeId });
+                _context.SaveChanges();
```

Comment:
lets always use `SaveChangesAsync`, async all db calls in all other methods, and rename method to `AddCompanionToEpisodeAsync`